### ISSUE TITLE
LuLu prevent qBittorrent start

### ISSUE THREAD

**Issue by elecenet** [objective-see/LuLu#437]
Hi! I'm using Mac OSX (Monterey 12.1) and noticed a strange interaction between LuLu and qBittorrent:

- if I start Qbit with LuLu active, it becomes irresponsive (Beachball of Doom, sometimes it can load the main screen) and need to shutdown the program.
- if LuLu is not active, or I disable it, the QBit starts and work perfectly. Then I can enable LuLu and all is OK until next restart.

There is an specific rule to allow Qbit in any address:any port within LuLu.

This behavior started months ago. I noticed today the same thing on the Tor browser, as an app or as "private window" in Brave browser. App need to be initiated with LuLu disabled, and then works.

**Comment by AlexGoodhart** [comment]
Yes, what's strange is that LuLu provides no pop-up alerting you to the application attempting to communicate, and it does not appear in the rules list.
